clairvoyante/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+python2.7 -m pip install intervaltree
```

Comment:
Instead of installing with pip, could intervaltree and blosc be moved to the meta.yaml requirements run section?

Reply:
I was doing that in an earlier version. However, there exists no intervaltree and blosc for pypy in conda, so I need to pip install intervaltree and blosc using for pypy anyhow. For a better consistency, I also used pip to install the two libs for python.

Reply:
@corburn Just a courtesy reminder, please advise how to proceed, thank you.

Reply:
@aquaskyline you could try pinging bioconda/core again.

Personally I'd expect a conda package to add commands to the PATH:
```bash
conda install clairvoyante
runDemo
```

From the recipe it looks like users would be expected to run:
```bash
conda install clairvoyante
pypa /path/to/conda/env/bin/clairvoyante/runDemo.py
```

I do not know where a conda recipe should copy a python package that is not using setuptools so it will be found on import. The following is still not correct, but I got as far as trying:
```yaml
package:
  name: clairvoyante
  version: 1.0

source:
  url: https://github.com/aquaskyline/Clairvoyante/releases/download/v1/Clairvoyante_v1.0.zip
  md5: 74b22669edcc08ee7be01b09f52569ee

build:
  noarch: python
  number: 0
  script:
    - mkdir -pv $STDLIB_DIR/site-packages
    - cp -rv clairvoyante dataPrepScripts $STDLIB_DIR/site-packages
  entry_points:
    - demoRun = clairvoyante.demoRun:Run
    #...

requirements:
  run:
    - python =2.7
    - pypy2.7
    - intervaltree
    - blosc
    - tensorflow
    - numpy
    - zlib

test:
  imports:
    - clairvoyante
  commands:
    - demoRun

about:
  home: 'https://github.com/aquaskyline/Clairvoyante'
  license: AGPLv3
  license_file: LICENSE.md
  summary: "A deep neural network based variant caller"
```

Reply:
@bioconda/core. Thanks for your suggestions,I've spent a few hours tweaking my tool as well as the recipe to make the best out of both, the main points are as follows:
1. In my tool, I have added an entry point that enables access to all the submodules, named `clairvoyante.py`. It's command-line accessible without a path.
2. I insist on using `build.sh` instead of putting everything in the `meta.yaml`.
3. I've tried installing `blosc` and `intervaltree` using conda. The libraries can be successfully installed, but not accessible. One way or another, I will have to install the two libraries for `pypy2.7`, so for consistency, I choose to do it the same for `python2.7`.
4. @corburn has suggested to put the scripts in $STDLIB_DIR, but I put them in $PREFIX/bin instead because they are programs instead of libraries.

I've tested locally and committed the updated recipe.